saw-core/prelude/Prelude.sawcore

```diff
@@ -793,20 +793,360 @@ maybe : (a b : sort 0) -> b -> (a -> b) -> Maybe a -> b;
 maybe a b f1 f2 m = Maybe__rec a (\ (m':Maybe a) -> b) f1 f2 m;
 
 
+--------------------------------------------------------------------------------
+-- Positive Natural Numbers
+
+data Pos : sort 0 where {
+    One : Pos;
+    Bit0 : Pos -> Pos;
+    Bit1 : Pos -> Pos;
```

Comment:
Perhaps obvious to folks in the know, but it might worth commenting that `Bit0` and `Bit1` represent numbers of the form `2*n` and `2*n + 1`, respectively, for some natural number `n`.